File: src/Backend/Curotec.Store/backend/Curotec.Store.Business/Models/Product.cs
namespace Curotec.Store.Business.Models
{
    public class Product: Entity
    {
        public string Name { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public decimal Value { get; set; }
        public DateTime CreatedAt { get; set; }
        public bool Active { get; set; }

        /* EF Relations */
        public Guid ContractorId { get; set; }
        public Contractor Contractor { get; set; } = default!;
    }
}

File: src/FrontEnd/Curotec-Web/src/app/product/models/product.ts
export interface Product {
  id: string;
  name: string;
  description: string;
  //image: string,
  //imagemUpload: string;
  value: number;
  CreatedAt: string;
  active: true;
  contractorId: string;
  nameContractor: string;
}

export interface Contractor {
  id: string;
  name: string;
}
